.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Report a problem to help us improve Sopel
+title: ''
+labels: Bug
+---
+
+Before submitting, please make sure you are running a supported, unmodified
+version of Sopel and that you have searched to see if this issue is
+already reported.
```

Comment:
Let's encourage testing against `master` if possible. A draft replacement for this paragraph:

```
Before reporting a bug, please search for existing reports. Checking both open
*and* closed issues is important, because your bug might already be fixed for
an upcoming release. If you can't find an existing report, please try to
reproduce the problem you found on our `master` branch first. Sometimes bugs
get fixed without needing a report!
```

Of course, the longer this is, the less likely it is that a user will actually read the whole thing… If you've an idea to make this more concise, I'm all ears/eyes.

Reply:
I make a point of _unmodified_ since a recent IRC report turned out to be because of hacking around inside core.